docs/3.0.0-beta.x/guides/slug.md

```diff
@@ -0,0 +1,107 @@
+# Create a slug system
+
+In this guide we will see how to create a slug system for your Post, Article or the Content Type you want.
```

Comment:
`This guide will explain how to create a slug system for a Post, Article or any Content Type you want.`